Description: fix include in cpu backend pulling unneeded bdeps
Author: Ghislain Antony Vaillant <ghisvail@gmail.com>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/backend/cpu/CMakeLists.txt
+++ b/src/backend/cpu/CMakeLists.txt
@@ -31,8 +31,6 @@
 MESSAGE(STATUS "FFTW Found ? ${FFTW_FOUND}")
 MESSAGE(STATUS "FFTW Library: ${FFTW_LIBRARIES}")
 
-INCLUDE("${CMAKE_MODULE_PATH}/FindGLEWmx.cmake")
-
 IF(APPLE)
     FIND_PACKAGE(LAPACK)
 ELSE(APPLE) # Linux and Windows
